src/context/Controller.js

```diff
@@ -0,0 +1,19 @@
+import React, { createContext, useState } from 'react';
+
+export const Context = createContext();
+export function Provider({ children }) {
+    const [values, setValues] = useState({});
+
+    function onControllerChange(name, value) {
+        const newState = {
+            ...values,
+            [name]: value,
+        };
```

Comment:
Where is the starting state?
I want some schema that can work without guessing what state parameters can get.
I'm not sure about touchX and touchY vars and I don't think we should keep state.
what do you think of just working simple without state? I don't have to keep the robot's location, it's a source of bugs.

Reply:
Leave it for now, I'll wait to hear from hardware how they want comms to work